rpc/Cargo.toml

```diff
@@ -33,8 +33,8 @@ rustc-hex = "1.0"
 rustc-serialize = "0.3"
 time = "0.1"
 tokio-core = "0.1.17"
-jsonrpc-core = { git = "https://github.com/paritytech/jsonrpc.git", tag = "v14.0.0" }
-jsonrpc-derive = { git = "https://github.com/paritytech/jsonrpc.git", tag = "v14.0.0" }
-jsonrpc-http-server = { git = "https://github.com/paritytech/jsonrpc.git", tag = "v14.0.0" }
-jsonrpc-ipc-server = { git = "https://github.com/paritytech/jsonrpc.git", tag = "v14.0.0" }
-jsonrpc-ws-server = { git = "https://github.com/paritytech/jsonrpc.git", tag = "v14.0.0" }
+jsonrpc-core = { git = "https://github.com/foriequal0/jsonrpc.git", branch = "fix/wait-race" }
```

Comment:
Why not fork jsonrpc under CodeChain-io and get a peer review for the fix?

Reply:
Okay. I've already made a PR to them, so I'll do that next time.